### PR TITLE
fix: seperate csp for admin page

### LINKED ISSUE #651: CSP cause Netlify CMS failed to open
## Description

On #634 we have CSP related pull-request for issue #611

It cause our CMS Netlify failed to open on https://www.wargabantuwarga.com/admin/

Preview Blank Page:

Screen Shot 2021-08-12 at 14 39 54

Error:

Screen Shot 2021-08-12 at 14 39 48

```
Content Security Policy: The page’s settings blocked the loading of a resource at https://identity.netlify.com/v1/netlify-identity-widget.js (“script-src”).
Content Security Policy: The page’s settings blocked the loading of a resource at https://unpkg.com/netlify-cms@%5E2.0.0/dist/netlify-cms.js (“script-src”).
```

## Expected Behaviour

Should not effected

## Additional Info

Our homepage also show error

Screen Shot 2021-08-12 at 14 43 59

```
Content Security Policy: The page’s settings blocked the loading of a resource at https://analytics.google.com/g/collect?v=2&tid=G-Q32QC5END5&gtm=2oe8b0&_p=1768343212&sr=1920x1080&_gaz=1&ul=en-us&cid=1694819148.1628219111&_s=1&dl=https%3A%2F%2Fwww.wargabantuwarga.com%2F&dt=Informasi%20Faskes%20%26%20Alkes%20untuk%20COVID-19%20%7C%20Warga%20Bantu%20Warga&sid=1628753583&sct=9&seg=1&en=page_view&ep.debug_mode=true (“default-src”).
```

### PR DESCRIPTION
**Pull request by asaadam**
Closes #651 
## Description

seperate csp header for admin page

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -4,18 +4,30 @@
...
+
 [[headers]]
-  for = "/_next/static/*"
+  for = "/admin/*"
   [headers.values]
+    X-Frame-Options = "DENY"
+    X-XSS-Protection = "1; mode=block"
+    X-Content-Type-Options = "nosniff"
+    Referrer-Policy = "same-origin"
+    Content-Security-Policy = "default-src 'self'; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' *.googletagmanager.com *.netlify.com unpkg.com analytics.google.com 'unsafe-eval'; img-src 'self' blob: data: https:; frame-ancestors 'none'; "
```
Comment (mazipan) [inline netlify.toml:25]: `media-library.cloudinary.com;` adn `cloudinary.com;` is required on CMS, it's used by upload image

Comment (asaadam) [reply netlify.toml]: okayy,,
please check it agian

**Review by mazipan — APPROVED** [review]
OK from me, let's wait @resir014 to take a look and test the Cloudinary part.
I have no access into Cloudinary

File: netlify.toml [hunk 2]
```diff
@@ -4,18 +4,30 @@
...
+
 [[headers]]
   for = "/*"
   [headers.values]
     X-Frame-Options = "DENY"
     X-XSS-Protection = "1; mode=block"
     X-Content-Type-Options = "nosniff"
     Referrer-Policy = "same-origin"
-    Content-Security-Policy = "default-src 'self'; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' *.googletagmanager.com *.netlify.com unpkg.com media-library.cloudinary.com analytics.google.com 'unsafe-eval'; frame-src 'self' cloudinary.com; img-src 'self' blob: data: https:; frame-ancestors 'none';"
+    Content-Security-Policy = "default-src 'self'; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' *.googletagmanager.com analytics.google.com media-library.cloudinary.com; frame-src 'self' cloudinary.com; img-src 'self' blob: data: https:; frame-ancestors 'none'; "
```
Comment (resir014) [inline netlify.toml:15]:
```suggestion
    Content-Security-Policy = "default-src 'self'; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' *.googletagmanager.com analytics.google.com; img-src 'self' blob: data: https:; frame-ancestors 'none';"
```

File: netlify.toml [hunk 3]
```diff
@@ -4,18 +4,30 @@
...
+
 [[headers]]
-  for = "/_next/static/*"
+  for = "/admin/*"
   [headers.values]
+    X-Frame-Options = "DENY"
+    X-XSS-Protection = "1; mode=block"
+    X-Content-Type-Options = "nosniff"
+    Referrer-Policy = "same-origin"
+    Content-Security-Policy = "default-src 'self'; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' *.googletagmanager.com *.netlify.com unpkg.com analytics.google.com 'unsafe-eval' media-library.cloudinary.com; frame-src 'self' cloudinary.com; img-src 'self' blob: data: https:; frame-ancestors 'none'; "
```
Comment (resir014) [inline netlify.toml:25]:
```suggestion
    Content-Security-Policy = "default-src 'self'; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' *.googletagmanager.com *.netlify.com unpkg.com analytics.google.com 'unsafe-eval' media-library.cloudinary.com; frame-src 'self' cloudinary.com; img-src 'self' blob: data: https:; frame-ancestors 'none';"
```

**Review by resir014 — DISMISSED** [review]
Now that we have separate CSP headers for /admin/*, we can remove the CSP rules meant for the admin page from `/*` and test if the CMS admin page still works.

**Review by resir014 — APPROVED** [review]
All tests are good on my end. Let's ship this! 🚢 

image

**Pull request merged**